cmd/minishift/cmd/start.go

```diff
@@ -144,16 +116,16 @@ func runStart(cmd *cobra.Command, args []string) {
 	proxyConfig := handleProxies()
 
 	machineConfig := cluster.MachineConfig{
-		MinikubeISO:      viper.GetString(isoURL),
-		Memory:           viper.GetInt(memory),
-		CPUs:             viper.GetInt(cpus),
-		DiskSize:         calculateDiskSizeInMB(viper.GetString(humanReadableDiskSize)),
-		VMDriver:         viper.GetString(vmDriver),
+		MinikubeISO:      viper.GetString(config.ISOUrl.Name),
```

Comment:
If we alias the import we might have something more understandable:

    viper.GetString(settings.ISOUrl.Name)

or 

    viper.GetString(startFlags.ISOUrl.Name)